lib/telemetry.ex

```diff
@@ -84,7 +84,8 @@ defmodule Logflare.Telemetry do
         tags: [:processor],
         unit: {:native, :millisecond}
       ),
-      counter("logflare.context_cache.busted.count")
+      counter("logflare.context_cache.busted.count"),
+      counter("logflare.context_cache.handle_record.count")
```

Comment:
Here we could record count of evicted entries by type.

Reply:
Added